configure.py

```diff
@@ -672,6 +678,9 @@ def _ffmpeg_setup(cfg):
             f"--cc={cfg.android_ndk_bin}{os.sep}{cfg.android_compiler}-clang",
         ]
     elif cfg.host == "iOS":
+        decoders += [
+            "aac_at",
+        ]
```

Comment:
What happened here with the AAC encoders and decoders?

Reply:
It makes use of the platform decoders instead of the ffmpeg ones to avoid licensing issue.

Reply:
Also mentionned in the commit message.